src/sdp/data.rs

```diff
@@ -1227,7 +1228,7 @@ impl fmt::Display for Sdp {
 impl fmt::Display for Session {
     fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
         write!(f, "v=0\r\n")?;
-        write!(f, "o=- {} 2 IN IP4 0.0.0.0\r\n", self.id)?;
+        write!(f, "o=str0m-{} {} 2 IN IP4 0.0.0.0\r\n", VERSION, self.id)?;
```

Comment:
Including the version here might be more hassle than it's worth. I've fixed some of the tests that fail as a consequence of this, but not all.